infra/docker/keycloak/themes/phrasea/login/resources/css/phrasea/login-phrasea.css

```
@@ -6,6 +6,10 @@
     background-size: cover;
     height: 100%;
     background: #f1f1f1;
+    background-image: url("##background-image##");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-attachment: fixed;
```

Discussed with @nmaillat 

Can you merge background properties into one:

```suggestion
    background: ##KC_LOGIN_CSS_BACKGROUND##;
```

with default value for `KC_LOGIN_CSS_BACKGROUND="#f1f1f1"`

So for custom background image we could set: `KC_LOGIN_CSS_BACKGROUND="url(https://static.com/pattern.jpg) #f1f1f1 no-repeat center center fixed"`